server/content/establishmentData.json

```diff
@@ -20,9 +19,8 @@
   },
   "793": {
     "name": "wayland",
-    "formattedName": "Wayland",
-    "prefix": "HMP",
-    "uuId": "b73767ea-2cbb-4ad5-ba22-09379cc07241",
+    "displayName": "HMP Wayland",
+    "searchName": "Wayland",
```

Comment:
tidying to make these obvious and removing extra logic around concatenating prefix and name